third_party/2/enum.pyi

```diff
@@ -16,7 +16,6 @@ class Enum(metaclass=EnumMeta):
     name = ...  # type: str
     value = ...  # type: Any
 
-class IntEnum(int, Enum):  # type: ignore
-    def __new__(cls: Type[_T], value: Any) -> _T: ...
+class IntEnum(int, Enum): ...
```

Comment:
I also removed `__new__` from int, so I don't think that makes a difference. Int comes first at runtime (https://github.com/python/cpython/blob/master/Lib/enum.py#L639), so putting Enum first here will probably break something.